<launch>

<node pkg="mw-exploration" type="explore_server" respawn="false" name="explore_server" output="screen">
  <!--机器人底盘坐标系-->
  <param name="robot_base_frame" value="base_link"/>
  <!--地图主题-->
  <param name="costmap_topic" value="/move_base/global_costmap/costmap"/>
  <!--地图更新，如果地图来源始终发布完整更新，即不提供此主题，则不需要-->
  <param name="costmap_updates_topic" value="/move_base/global_costmap/costmap_updates"/>
  <!--指定是否发布可视化边界-->
  <param name="visualize" value="true"/>
  <!--计算新边界并重新考虑目标的速率(Hz)-->
  <param name="planner_frequency" value="0.77"/>
  <!--时间以秒为单位，当机器人在 progress_timeout 内没有任何进展时，将放弃当前目标-->
  <param name="progress_timeout" value="30.0"/>
  <!--用于加权边界,这个乘法参数影响边界权重的边界潜在分量（到边界的距离）-->
  <param name="potential_scale" value="1.9"/>
  <!--用于加权边界，乘法参数影响边界权重（边界大小）的边界增益分量-->
  <param name="gain_scale" value="0.7"/>
  <!--用于加权边界，乘法参数影响信息增益的增益分量-->
  <param name="information_scale" value="0.9"/>
  <!--用于加权边界，该乘法参数影响边界权重的边界方向分量-->
  <param name="orientation_scale" value="1.1"/>
  <!--变换机器人姿势时使用的变换容差-->
  <param name="transform_tolerance" value="0.3"/>
  <!--将边界视为探索目标的最小边界大小,以米为单位-->
  <param name="min_frontier_size" value="10.0"/>
  <!--前沿目标点的类型 initial/centroid/middle-->
  <param name="frontier_type" value="centroid"/>
  <!--信息增益检测半径，以米为单位-->
  <param name="information_r" value="3.0"/>
  
</node>

<node pkg="mw-exploration" type="plugin_client" respawn="false" name="plugin_client" output="screen">

</node>

</launch>
